### PR TITLE
CA-186 Adds KMS Interface Endpoint.

### PR DESCRIPTION
**Pull request by fvazquez-caylent**
* Adds KMS VPC Interface Endpoint.

### CODE REVIEW

File: examples/minimal/sg_web.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module "sg_vm_web" {
-  source = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  source = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.2.0"
```
Comment (souza-dan) [inline examples/minimal/sg_web.tf:2]: Latest release of the security groups module is 1.0.0:
https://github.com/Datatamer/terraform-aws-security-groups/releases
```suggestion
  source = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
```

File: vpc-endpoints.tf [hunk 2]
```diff
@@ -58,3 +66,19 @@ resource "aws_security_group" "logs_interface_endpoint" {
...
+  name        = format("%s-%s", var.name_prefix, "kms-interface-endpoint-sg")
+  description = "Security Group to be attached to the KMS Endpoint interface, which allows TCP traffic to the KMS service."
+  vpc_id      = module.vpc.vpc_id
+
+  ingress {
+    description = "KMS API"
+    from_port   = 443
+    to_port     = 443
+    protocol    = "TCP"
+    cidr_blocks = [var.compute_subnet_cidr_block]
```
Comment (souza-dan) [inline vpc-endpoints.tf:81]: Would the application and data subnets benefit from being able to use this endpoint as well?

Comment (franco-caylent) [reply vpc-endpoints.tf:81]: If we decide to implement KMS encryption for tamr-vm yes, not so sure about RDS but i'd suppose yes.

**Review by franco-caylent — CHANGES_REQUESTED** [review]
LGTM, please update the security groups module before merging

**Review by souza-dan — APPROVED** [review]
LGTM

**Review by franco-caylent — APPROVED** [review]
LGTM

**Pull request merged**